### PR TITLE
A couple of fixes for the NAPALM NTP modules

### PR DESCRIPTION
**Pull request by mirceaulinic**
### What does this PR do?

Adds a couple of improvements for the NAPALM modules:
- documentation updates in napalm proxy, napalm_ntp execution module and netntp state
- import check in napalm_ntp module
- new function in napalm_ntp module: stats(), based on the newest NAPALM lib updates
- test option in the state
- return config changes in the state

### CODE REVIEW

File: salt/modules/napalm_ntp.py [hunk 1]
```diff
@@ -42,7 +53,17 @@
...
+
+    """
+    NAPALM library must be installed for this module to work.
+    Also, the key proxymodule must be set in the __opts___ dictionary.
+    """
+
+    if HAS_NAPALM and 'proxy' in __opts__:
+        return __virtualname__
+    else:
+        return (False, 'The mdoule NTP cannot be loaded: \
```
Comment (rallytime) [inline salt/modules/napalm_ntp.py]: `mdoule` should be `module`.

File: salt/modules/napalm_ntp.py [hunk 2]
```diff
@@ -42,7 +53,17 @@
 
 
 def __virtual__():
-    return True
+
+    """
```
Comment (rallytime) [inline salt/modules/napalm_ntp.py]: Single quotes here, instead of double, here as well.

File: salt/modules/napalm_ntp.py (hunk 1 above)
Comment (mirceaulinic) [inline salt/modules/napalm_ntp.py]: Fixed that too. Thanks!

**Comment by rallytime** [comment]
Thanks @mirceaulinic!

**Pull request merged**